di_website/static/css/di_website.css

```diff
@@ -191,3 +191,43 @@ select optgroup {
 .is-typeset .cta {
   margin-top: 5rem;
 }
+
+.body--leaf .accordion__heading {
+  background-color: #109E67;
+}
+
+.body--poppy .accordion__heading {
+  background-color: #e84439;
+}
+
+.body--bluebell .accordion__heading {
+  background-color: #0089CB;
+}
+
+.body--rose .accordion__heading {
+  background-color: #C2145B;
+}
+
+.body--sunflower .accordion__heading {
+  background-color: #F49B20;
+}
+
+.body--marigold .accordion__heading {
+  background-color: #EB632B;
+}
+
+.body--lavendar .accordion__heading {
+  background-color: #89328F;
+}
+
+.accordion-spacing .accordion__list-item:first-child {
```

Comment:
In addition to this, give the last child `margin-bottom: 2em` ... otherwise it looks cramped in some non-section usages. The one below is for the AudioVisual page type. Perhaps only add this margin for when the accordion doesn't have it's own section ... I think the distinguishing class for a bottom margin is `wysiwyg`.

`.wysiwyg .accordion-spacing .accordion__list-item:last-child` - maybe

<img width="795" alt="image" src="https://user-images.githubusercontent.com/5672438/194231409-5b0fb2ed-170d-4fa2-a589-481f4d5a7b0a.png">